test/functional/Tactic.hs

```diff
@@ -106,7 +106,8 @@ tests = testGroup
   , goldenTest "GoldenShowCompose.hs"       2 15 Auto ""
   , goldenTest "GoldenShowMapChar.hs"       2 8  Auto ""
   , goldenTest "GoldenSuperclass.hs"        7 8  Auto ""
-  , goldenTest "GoldenApplicativeThen.hs"   2 11 Auto ""
+  -- TODO(sandy): ignored, as it's slow and flakey in CI
+  -- , goldenTest "GoldenApplicativeThen.hs"   2 11 Auto ""
```

Comment:
A nice alternative to comment the code is to use tasty `ignoreBecause`. It still is being compiled and reported in the test suite.
Feel fee to cherry-pick/adapt my commit in hls-plugin: e54e7dc